bite-size-notes/week-8-d-Probability and Recursive Bayesian Filtermd.md

```diff
@@ -0,0 +1,210 @@
+
+$$
+p(x) = \det(2\pi\Sigma)^{-\frac{1}{2}} \exp \left( -\frac{1}{2} (x - \mu)^T \Sigma^{-1} (x - \mu) \right) \tag{2}
+$$
+
+
+where $\mu$ is the mean vector, $\Sigma$ is a positive semidefinite and symmetric matrix called the covariance matrix; $T$ denotes the transpose of the vector.
+
+The joint distribution of two random variables $\mathbf{X}$ and $\mathbf{Y}$ is:
+
+$$
+p(x, y) = p(X = x, Y = y) \tag{3}
+$$
+
+If $\mathbf{X}$ and $\mathbf{Y}$ are independent, we have:
+
+$$
+p(x, y) = p(x)p(y) \tag{4}
+$$
+
+Assume that the value of $\mathbf{Y}$ is known to be $y$, and we want to know the probability of $\mathbf{X}$ being $x$ given this fact, expressed as
+
+$$
+p(x|y) = p(X = x | Y = y) \tag{5} 
+$$
+
+This is called the conditional probability. If $p(y) > 0$, the conditional probability is defined as
+
+$$
+p(x|y) = \frac{p(x,y)}{p(y)} \tag{6} 
+$$
+
+If $\mathbf{X}$ and $\mathbf{Y}$ are independent, then
+
+$$
+p(x|y) = \frac{p(x)p(y)}{p(y)} = p(x) \tag{7} 
+$$
+
+In other words, if $\mathbf{X}$ and $\mathbf{Y}$ are independent, then $\mathbf{Y}$ provides no information about the value of $\mathbf{X}$.
+
+### Theorem of Total Probability
+#### discrete case：
+$$
+p(x) = \sum_y p(x|y)p(y) \tag{8} \quad  
+$$
+#### continuous case：
+$$
+p(x) = \int p(x|y)p(y)dy \tag{9} \quad 
+$$
+### Bayes Rule
+Bayes' Rule links the conditional probability $p(x|y)$ with the "reverse" probability $p(y|x)$, requiring $p(y) > 0$:
+#### discrete case：
+$$
+p(x|y) = \frac{p(y|x)p(x)}{p(y)} = \frac{p(y|x)p(x)}{\sum_{x'} p(y|x')p(x')} \tag{10} \quad 
+$$
+#### continuous case：
+$$
+p(x|y) = \frac{p(y|x)p(x)}{p(y)} = \frac{p(y|x)p(x)}{\int p(y|x')p(x')dx'} \tag{11} \quad 
+$$
+
+If $x$ is a value inferred from $y$, the probability $p(x)$ is called the prior probability distribution. Here, $y$ is referred to as data, which could also be sensor measurements. The distribution $p(x)$ summarizes all prior information about $x$ before observing $y$. The probability $p(x|y)$ is known as the posterior probability distribution. As shown in equations (10) and (11), Bayes rule provides a convenient method to calculate the posterior probability $p(x|y)$ by using the reverse conditional probability $p(y|x)$ and the prior probability $p(x)$.
+
+## Main body
+### Recursive Bayesian Filter
+- Predict
+
+$$
+P(x_k | y_{1:k-1}) = \int_{x_{k-1}}P(x_k | x_{k-1}) \cdot P(x_{k-1} | y_{1:k-1})d x_{k-1}
+$$
+
+$$
+P(x_k, x_{k-1} | y_{1:k-1}) = P(x_k | x_{k-1}, y_{1:k-1}) \cdot P(x_{k-1} | y_{1:k-1})
+$$
+
+$$
+=P(x_k, x_{k-1}) \cdot P(x_{k-1} | y_{1:k-1})
+$$
+
+$$
+P(x_k | y_{1:k-1}) = \int_{x_{k-1}}P(x_k , x_{k-1}| y_{1:k-1}) dx_{k-1}
+$$
+
+- Update
+
+$$
+P(x_k | y_{1:k}) \propto P(y_k | x_k) \cdot P(x_k | y_{1:k-1}) 
+$$
+
+$$
+P(x_k | y_{1:k}) = P(x_k | y_k, y_{1:k-1}) 
+$$
+
+$$
+\propto P(y_k | x_k, y_{1:k-1}) \cdot P(x_k | y_{1:k-1}) 
+$$
+
+$$
+\propto P(y_k | x_k) \cdot P(x_k | y_{1:k-1})
+$$
+- ![Markov chain](figs/pdf.jpg)
```

Comment:
The figure doesn't show up in the preview, but that may be because the file isn't added to the repo yet

Reply:
I've re-added it to the repo, so it should be fixed.